component/modal/modal.css

```diff
@@ -0,0 +1,571 @@
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-auto-flow: column;
+    grid-template-columns: 200px 1fr; 
+ 
+    grid-template-areas: 
+    "nav nav "
+    "pitch pitch"
+   "aside main"
+   "aside main"
+   "aside main"
+   "footer footer"
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+    
+
+}
+
+nav h2{
+    position: relative;
+    top: 35px;
+    font-size: 2rem;
+    line-height: 2.5rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto; 
+    display: flex;
+  gap: 15px;
+  position: relative;
+  bottom: 35px;
+  padding-inline-end: 0.7rem;
+ 
+}
+
+li{
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+   
+    grid-area: main;
+    font-size: 1.2rem;
+    padding-left: 7rem;
+    padding-right: 7rem;
+    
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+aside {
+
+    background: #111827;
+}
+
+aside label{
+    color: var(--black);
+}
+
+aside #hamburger{
+grid-area: ham;
+display: none;
+
+}
+
+
+h3{
+    font-size: 1.5rem;
+    /* line-height: 2rem; */
+    letter-spacing: 0.1rem;
+ }
+
+ .impact{
+     font-style: italic;
+     color: var(--secondary-color);
+     
+ }
+
+ 
+ 
+
+.component-text{
+    grid-area: pitch;
+    font-size: 2rem;
+    margin: auto;
+    letter-spacing: 0.1rem;
+}
+
+
+hr{
+    border-top: 1px solid var(--primary-color); 
+  width: 50rem;
+}
+
+.highlight{
+    color: var(--secondary-color);
+}
+
+iframe{
+    padding-top: 1rem;
+}
+
+
+
+
+.medium{
+    width: 4.5rem;
+    height: 4.5rem;
+    margin-left: 2rem; 
+}
+
+.large{
+    width: 5.5rem;
+    height: 5.5rem;
+    margin-left: 2rem; 
+}
+
+.bold{
+    font-weight: 600;
+    font-size: 1.5srem;
+    padding-top: 3.5rem;
+}
+
+.modal-container .btn {
+    color: var(--secondary-color);
+    background-color: var(--primary-color);
+    margin: 0.5rem;
+    border: none;
+    font-size: 1rem;
+    padding: 0.5rem;
+    border-radius: 0.3rem;
+    font-weight: 600;
+}
+
+.modal {
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
+}
+
+.modals {
+    background-color: var(--secondary-color);
+    border-top: 1px solid var(--primary-color);
+    padding: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.alert-modal {
+    background-color: white;
+    color: var(--secondary-color);
+    padding: 1rem;
+    border: 1px solid var(--primary-color);
+    box-shadow: 0 0 5px hotpink;
+}
+
+.unorder-li-group {
+    list-style: none;
+    padding: 0.5rem;
+}
+
+
+

```

Comment:
Extra space can increase the time complexity

Reply:
agree